docs/basics/control_flow_commands.md

```diff
@@ -0,0 +1,117 @@
+
+The `cont()` command continues the execution.
+
+!!! ABSTRACT "Function Signature"
+    ```python
+    d.cont()
+    ```
+
+!!! INFO "Script execution"
+    Please note that, after resuming execution of the tracee process, the script will continue to run. This means that the script will not wait for the process to stop before continuing with the rest of the script. If the next command is a **libdebug** command that requires the process to be stopped, the script will then wait for a [stopping event](../../stopping_events/stopping_events) before executing that command.
+
+    For example, in the following script, **libdebug** will not wait for the process to stop before checking d.dead. To change this behavior, you can use the `wait()` command right after the `cont()`.
+    ```python
+    from libdebug import debugger
+
+    d = debugger("program_that_dies_tragically")
+
+    d.run()
+
+    d.cont()
+
+    if d.dead:
+        print("The program is dead!")
+
+    ```
+
+### :material-clock-alert-outline: The `wait()` Method
+
+The `wait()` command is likely the most important in **libdebug**. Loved by most and hated by many, it instructs the debugger to wait for a [stopping event](../../stopping_events/stopping_events) before continuing with the execution of the script.
+
+!!! ABSTRACT "Example"
+    In the following script, **libdebug** will wait for the process to stop before printing "provola".
+    ```python
+    from libdebug import debugger
+
+    d = debugger("program_that_dies_tragically")
+
+    d.run()
+
+    d.cont()
+    d.wait()
+
+    print("provola")
+    ```
+
+### :material-stop: Interrupt
+You can manually issue a stopping signal to the program using the `interrupt()` command. Clearly, this command is issued as soon as it is executed within the script.
+
+!!! ABSTRACT "Function Signature"
+    ```python
+    d.interrupt()
+    ```
+
+## :material-debug-step-out: Finish
+
+The `finish()` command continues execution until the current function returns or a breakpoint is hit. In other debuggers, this command is known as "step out".
+
+!!! ABSTRACT "Function Signature"
+    ```python
+    d.finish(heuristic='backtrace')
+    ```
+
+!!! WARNING "Damn heuristics!"
+    The `finish()` command uses heuristics to determine the end of a function. While **libdebug** allows to choose the heuristic, it is possible that none of the available options work in some specific cases. (e.g. tail-calls, non-returning calls).
+
+### Available Heuristics
+The `finish()` command allows you to choose the heuristic to use. If you don't specify any, the `"backtrace"` heuristic will be used. The following heuristics are available:
+
+| Heuristic | Description |
+|-----------|-------------|
+| `backtrace` | The `backtrace` heuristic uses the return address on the stack function frame to determine the end of the function. This is the default heuristic but may fail in case of broken stack, rare execution flows, and obscure compiler optimizations. |
```

Comment:
on the stack function frame -> on the function stack frame

Reply:
[9e99adc](https://github.com/libdebug/libdebug/pull/129/commits/9e99adc7c4f740849576a6ec29d1c746c4e2a5ef)